client/src/components/review.jsx

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import moment from 'moment';
 
-const Review = (props) => ( // if a comment is more than 5 lines long, use the 'Show more' button
-  <div>
-    {/* {console.log('Props in Review:', props)} */}
+// note for later: if a comment is more than 5 lines long, hide the rest use a 'Show more' button
+
+const Review = (props) => {
+  // console.log('Props in Review:', props);
+
+  const CommentWithBoldSearchTerm = () => {
+    // subcomponent: current search term should be bold in each filtered review (for now, only works for the first occurrence of that word in each review)
+
+    let comment = props.review.comment;
+    let term = props.currentSearchTerm;
+    let occurrenceIndex = comment.toLowerCase().indexOf(term);
+    let isCapitalized = comment.charAt(occurrenceIndex) === comment.charAt(occurrenceIndex).toUpperCase();
+    isCapitalized ? term = term[0].toUpperCase() + term.slice(1) : term;
+
+    return (
+      <div>
+        {comment.slice(0, occurrenceIndex)}<strong>{term}</strong>{comment.slice(occurrenceIndex + term.length)}
+      </div>
+    );
+  };
+
```

Comment:
Rather than returning JSX here, could this JSX be stored in a variable, and then the value of that variable be iteratively changed for every occurrence of the search term?
(Also, not sure if you knew -- I just learned this recently: indexOf accepts a starting point for the search, so you can set it to the index of the previous occurrence + 1, maybe.)